composer.json

```diff
@@ -43,7 +44,8 @@
       "includes/rest-api"
     ],
     "psr-4": {
-      "Automattic\\WooCommerce\\": "src/"
+      "Automattic\\WooCommerce\\": "src/",
+      "Automattic\\WooCommerce\\Vendor\\League\\Container\\": "vendor/league/container/"
```

Comment:
I'm not sure about this one, though, it does raise an interesting problem with PSR-4 namespaces and this method of renaming. I'm wondering if we should also correct the namespace in the `composer.json` for the dependency that way Composer knows what the namespace is when generating it?

Reply:
You mean changing `autoload` and `autoload-dev` in `vendor/league/container/composer.json`? Why not, if only for consistency (doesn't seem to be needed or am I missing something?)

Reply:
The problem I have with putting this PSR-4 mapping here is that it doesn't feel like something that belongs in the root `composer.json` since we're declaring a new namespace for code that already has a different namespace declaration elsewhere. Since we've renamed the namespace inside the package vendor files I believe we should do the same in the package's `composer.json`. 

For context, the next version of the jetpack-autoloader we utilize will support PSR-4 namespacing in development, which will use that declared namespace. If we don't change the package name there too then it will cause problems!

Reply:
Just making sure you didn't miss this reply @Konamiman!

Reply:
Sorry I think I wasn't clear enough. After you wrote that comment I updated the `prefix-vendor-namespaces.sh` script so that it also changes the namespace inside `vendor/league/container/composer.json`. This is what you meant by "also correct the namespace in the composer.json for the dependency", right?